SAMv1.tex

```diff
@@ -304,6 +307,25 @@ \subsection{The header section}
 \end{longtable}
 \end{center}
 
+\subsubsection{Defined sub-sort terms}\label{sec:sub-sort}
+
+While sub-sort permits implementation dependent sorting, some terms are defined to have specific meanings.
+
+\begin{description}
+\item[lexicographical] sort order is defined as a character based dictionary sort with the character order as defined by the POSIX C locale.
+  For example ``abc'', ``abc17'', ``abc5'', ``abc59'' and ``abcd'' are in lexicographical order.
+
+\item[natural] sort order is an ordering of strings in the lexicographical order, except that multi-digit natural numbers (non-negative integers including leading zeros) are ordered as a single character with value equal to the numerical value, slotting into the collation order in the same location as single-digit numbers.  Numeric ties caused through leading zeros are broken with more-zeroes coming before fewer-zeros.  Apparent negative and fractional values are not treated as part of the number, with `\texttt{-}' and `\texttt{.}' being interpreted as all other single characters for collation purposes.
```

Comment:
Thanks for adding the stuff about `-` and `.` just being ordinary characters. Alas it's still pretty murky. Here's another attempt that might provide some more inspiration…

\item[natural] sort order is similar to lexicographical order except that runs of adjacent digits are considered to be numbers embedded within the text string, ordered numerically when compared to each other and ordered as single digits when compared to the surrounding non-digit characters. Runs that differ only in the number of leading zeros (thus are numerically tied) are ordered by more-zeros coming before fewer-zeros. The characters `-` and `.` are considered as ordinary characters, so apparently negative or fractional values are not treated as part of an embedded number.

Reply:
Happy to go with that.